docs/csharp/programming-guide/statements-expressions-operators/how-to-define-value-equality-for-a-type.md

```diff
@@ -12,27 +12,31 @@ ms.assetid: 4084581e-b931-498b-9534-cf7ef5b68690
 ---
 # How to define value equality for a class or struct (C# Programming Guide)
 
-When you define a [record](../classes-and-structs/records.md), the compiler automatically implements value equality. When you define a class or struct, you decide whether it makes sense to create a custom definition of value equality (or equivalence) for the type. Typically, you implement value equality when objects of the type are expected to be added to a collection of some sort, or when their primary purpose is to store a set of fields or properties. You can base your definition of value equality on a comparison of all the fields and properties in the type, or you can base the definition on a subset.
+When you define a [record](../classes-and-structs/records.md), the compiler automatically implements value equality. When you define a class or struct, you decide whether it makes sense to create a custom definition of value equality (or equivalence) for the type. Typically, you implement value equality when you expect to add objects of the type to a collection, or when their primary purpose is to store a set of fields or properties. You can base your definition of value equality on a comparison of all the fields and properties in the type, or you can base the definition on a subset.
```

Comment:
I'm wondering if this should be stronger:

```suggestion
[Records](../classes-and-structs/records.md), the compiler automatically implement value equality. Consider defining a `record` instead of a `class` when your type models data and should implement value equality. When you define a class or struct, you decide whether it makes sense to create a custom definition of value equality (or equivalence) for the type. Typically, you implement value equality when you expect to add objects of the type to a collection, or when their primary purpose is to store a set of fields or properties. You can base your definition of value equality on a comparison of all the fields and properties in the type, or you can base the definition on a subset.
```

Reply:
I made a few small changes, does this revision look good to you?

Reply:
I really like that @tdykstra That's perfect.